.github/workflows/ci.yaml

```
@@ -105,4 +105,4 @@ jobs:
           
 
 
-        
\ No newline at end of file
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)